mrp_subcontracting_purchase/demo/0001_demo_product_category.xml

```diff
@@ -0,0 +1,5 @@
+<odoo>
```

Comment:
Why do you put a number of the file names?

Reply:
The proper name is `product_category_demo.xml`.

Reply:
> The proper name is `product_category_demo.xml`.

Files name are changed.